packages/core/src/model-types.ts

```diff
@@ -24,24 +23,22 @@ export type AzureOpenAiEmbeddingModel = LiteralUnion<
  * GCP Vertex AI models for chat completion.
  */
 export type GcpVertexAiChatModel = LiteralUnion<
-  'gemini-1.0-pro' | 'gemini-1.5-pro' | 'gemini-1.5-flash' | 'chat-bison'
+  'gemini-1.0-pro' | 'gemini-1.5-pro' | 'gemini-1.5-flash'
 >;
 
 /**
  * AWS Bedrock models for chat completion.
  */
 export type AwsBedrockChatModel = LiteralUnion<
-  | 'amazon--titan-text-express'
-  | 'amazon--titan-text-lite'
   | 'anthropic--claude-3-haiku'
   | 'anthropic--claude-3-opus'
   | 'anthropic--claude-3-sonnet'
   | 'anthropic--claude-3.5-sonnet'
 >;
 
 /**
- * All available models for chat completion.
+ * Aicore Opensource models for chat completion.
```

Comment:
```suggestion
 * AI Core open source models for chat completion.
```

[req] please check in other places as well.